src/app_service/service/chat/service.nim

```diff
@@ -471,3 +471,24 @@ QtObject:
     except Exception as e:
       error "error while creating group chat", msg = e.msg
 
+  proc getMembers*(self: Service, communityID, chatId: string): seq[ChatMember] =
+    try:
+      var realChatId = chatId.replace(communityID, "")
+      let response = status_chat.getMembers(communityID, realChatId)
+      result = @[]
+      for (id, memberObj) in response.result.pairs:
+        var member = toChatMember(memberObj)
+        member.id = id
+        result.add(member)
+
+      let myPubkey = singletonInstance.userProfile.getPubKey()
+      # Sort to make yourself as the first result
```

Comment:
shall we have a secondary sort to avoid random sort afterward?

Do u think it would be better to return a tuple: ChatMember, seq[ChatMember]

where the first is yourself?

Reply:
I'm not sure what you mean. Like return the current user as well as the `seq`, so that others can do another sort?